.ci/jenkins/Jenkinsfile.prod.nightly

```diff
@@ -67,13 +68,6 @@ pipeline {
                     env.QUARKUS_VERSION = PME_BUILD_VARIABLES['quarkusVersion'].replaceAll("\\{\\{.*\\}\\}", env.QUARKUS_VERSION_COMMUNITY)
                     env.QUARKUS_PLATFORM_VERSION = PME_BUILD_VARIABLES['quarkusPlatformVersion'].replaceAll("\\{\\{.*\\}\\}", env.QUARKUS_VERSION_COMMUNITY)
 
-                    // export images registries
```

Comment:
For the records, those are hardcoded images registries and they pointed to deprecated quay.io/kiegroup nightly, like:

dataIndexImageRegistry=quay.io/kiegroup/kogito-data-index-ephemeral-nightly:latest
swfBuilderImageRegistry=quay.io/kiegroup/kogito-swf-builder-nightly:latest
swfDevModeImageRegistry=quay.io/kiegroup/kogito-swf-devmode-nightly:latest
operatorImageRegistry=quay.io/kiegroup/kogito-serverless-operator-nightly:latest